apps/redi-talent-pool/src/components/organisms/jobseeker-profile-editables/EditableProfessionalExperience.tsx

```diff
@@ -94,9 +95,9 @@ export function EditableProfessionalExperience({
                 </span>
               </div>
               <Content style={{ marginTop: '-0.5rem' }}>
-                {item.company ? (
-                  <p style={{ color: '#979797' }}>{item.company}</p>
-                ) : null}
+                <p style={{ color: '#979797' }}>
+                  {locationDetails(item?.company, item?.city, item?.country)}
+                </p>
```

Comment:
What do you think about keeping `item.company` condition to render this paragraph, as it is? As far as I understand from the Loom video, we're adding these location details (city and country) to help jobseekers be more specific about their experience and also employers to have a better understanding and easier lookup of the company. I'd say, the company is still the main piece of information in this paragraph so, without it, just the city and country wouldn't make much sense.

I know the company is optional in the ExperienceRecord type, which we can also discuss but for now, I guess it makes some sense to not show anything if the company name is not given. Let me know what you think!